packages/graphic-walker/src/components/pivotTable/index.tsx

```diff
@@ -41,7 +43,7 @@ interface PivotTableProps {
     draggableFieldState: DeepReadonly<DraggableFieldState>;
     visualConfig: IVisualConfig;
 }
-const PivotTable: React.FC<PivotTableProps> = (props) => {
+const PivotTable: React.FC<PivotTableProps> = observer((props) => {
```

Comment:
Component wrappers such as `memo()`, `forwardRef()`, `observer()` extract the name of the first argument as the display name of the return component, so using an anonymous FC as the argument of `observer()` may set name of `PivotTable` to "observerComponent" instead of "PivotTable", which will lead to a bad practice of debugging and troubleshooting.

Reply:
addressed in 80fac50750876e62f161c24cfd76cdd21eaff80c